.github/workflows/release.yml

```diff
@@ -0,0 +1,103 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+  workflow_dispatch:
+    inputs:
+      skip_code_checks:
+        description: 'Skip code checks'
+        required: true
+        default: true
+        type: boolean
```

Comment:
The workflow runs on push of a tag matching `v*.*.*` _or_ when run manually. If it is run manually, you can skip the time-consuming code checks with this input.